gdx/src/com/badlogic/gdx/graphics/g2d/ParticleEffect.java

```diff
@@ -226,7 +226,11 @@ public BoundingBox getBoundingBox () {
 		return bounds;
 	}
 
+	/**Scales all the parameters of all the emitters in this effect. The original values are not
+	 * recovered when {@link #reset()} is called, unless this effect came from a {@link ParticleEffectPool},
+	 * in which case it uses the pool's prototype effect to restore the original values.*/
 	public void scaleEffect (float scaleFactor) {
+		// If this method is ever modified, 

```

Comment:
That was intentional. It's kind of a hacky method. If anybody ever changes the way it works, I figure it wouldn't be obvious that the reset method in the subclass would also need to be updated.